pulp_python/app/pypi/views.py

```diff
@@ -214,7 +214,7 @@ def parse_url(link):
         packages = (parse_url(link) for link in links)
         return StreamingHttpResponse(write_simple_detail(package, packages, streamed=True))
 
-    @extend_schema(summary="Get package simple page")
+    @extend_schema(operation_id="pypi_simple_package_read", summary="Get package simple page")
```

Comment:
Is there any use in creating api docs for the pypi interface? In pulp_container, for example we deactivate the generation for the registry api completely. The api's are defined elsewhere and we can only get in trouble if there is any difference.

Reply:
Probably not, but I do use the generated schema in the tests.

Reply:
Well, we need it then i guess.